packages/tenant-process/src/services/readModelService.ts

```diff
@@ -566,6 +568,35 @@ export function readModelServiceBuilder(
         "data.state": delegationState.active,
       });
     },
+    async getDelegationById(
```

Comment:
I would make this method specific for Active consumer delegation so that it does not take `kind` or `state` as parameters - you can also reuse the `getDelegation` utility to avoid rewriting similar code. See the dual method for producer delegations above https://github.com/pagopa/interop-be-monorepo/pull/1219/files#diff-c572b9bb95c4fcf3227c7b8343252d56aedb4e17e3f5d299bbfe69068a41f5f6R562